### ISSUE TITLE
Publish & tag 0.4.0-alpha

### ISSUE THREAD

**Issue by mmagician** [arkworks-rs/snark#369]
## Summary

To make sure we don't try to publish/tag twice, this is a separate task. 

Once #368 is merged, the following should still be done:
tag commits on master & publish both crates with:

```
git tag v0.4.0-alpha.1
git push origin v0.4.0-alpha.1
cargo release publish
```

**Comment by mmagician** [comment]
https://github.com/arkworks-rs/snark/tree/v0.4.0-alpha.1
